### PR TITLE
Step one for issue #704

### PR DESCRIPTION
**Pull request by tarhata**
I added some tests to test the new methods, but wasn't sure where to put it so I made a new file for it. I feel like it's not written correctly, so let me know your thoughts.

Everything else wasn't too bad--just needed to figure out how to change the code of the admin pages to reflect the new methods.

### CODE REVIEW

File: sjfnw/grants/tests/test_givingprojectgrants.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+class NewGivingProjectGrant(BaseGrantTestCase):
+  """ Test giving project grant methods:
+      yearend due, total_amount, and grant_length
+  """
+
+  def setUp(self):
+    super(NewGivingProjectGrant, self).setUp()
+
+  def test_minimum_grant_information(self):
+    """ Verify that only minimum information input """
```
Comment (aisapatino) [inline sjfnw/grants/tests/test_givingprojectgrants.py]: missing part of the sentence i think. (i think `test_minimum_grant_information` is descriptive enough, so you can either take out the docstring or add to it, whichever you prefer)

**Comment by aisapatino** [comment]
This looks great! The tests look good to me.

Since you'll be working on this one in several steps, it makes sense to merge into a different branch than `master` - `master` should only have code that is ready to deploy. (For example, if we need to get a bug fix out ASAP, that will be complicated to do if master has code that shouldn't be deployed along with it). So we should wait to merge this feature into `master` until it's done and ready to go live.

However, we need a branch for you to merge PRs into and branch off of when starting a new part. I made a new branch `multi-year-grants` for this feature. So when you open your next PR, select that one instead of `master`. Ignore `master` and use `multi-year-grants` instead while working on this feature. 

For this one, I'll just close the PR without merging, and do the merge into `multi-year-grants`.

(Let me know if that doesn't make sense or if you have any questions :) )

**Comment by tarhata** [comment]
Makes sense. I'll push my code into `multi-year-grants` instead. Having a hard time with the YER stuff, so I'll probably plea for some help.

**Comment by aisapatino** [comment]
Just to clarify - don't push directly to `multi-year-grants`, but open your PR against it. Like this
screen shot 2015-05-14 at 9 00 33 am

(If that whole branching setup doesn't make sense let me know and I'll draw a diagram that should make more sense). `multi-year-grants` already has this code in it, so you can merge step two in when you're done, etc.

Definitely ask for help. Go ahead and email me with whatever you're stuck on and I'll see what I can do! Or if you have something in progress, you can open a PR and ask questions there. Whatever's easier.

**Pull request closed without merging**